src/api/courses.js

```diff
@@ -211,15 +231,50 @@ router.post(
   ],
   safeAsync(async (req, res, _next) => {
     const { name, shortcode } = matchedData(req)
+    const { isUnlisted } = req.body
+    const { questionFeedback } = req.body
     const course = Course.build({
       name,
       shortcode,
+      isUnlisted,
+      questionFeedback,
     })
     const newCourse = await course.save()
     res.status(201).send(newCourse)
   })
 )
 
+// Modify a queue for a course
+router.patch(
+  '/:courseId',
+  [
+    requireCourseStaff,
+    requireCourse,
+    check('name').exists(),
+    check('shortcode').exists(),
+    check('isUnlisted').isBoolean(),
+    check('questionFeedback').isBoolean(),
+    failIfErrors,
+  ],
+  safeAsync(async (req, res, _next) => {
+    const courseId = res.locals.course.dataValues.id
+    const { course } = res.locals
+    const data = matchedData(req)
+    await course.update({
+      name: data.name !== null ? data.name : undefined,
```

Comment:
I'm not entirely sure if we want to allow users to modify the names of their own courses, or to update their shortcodes. @wadefagen do you have any thoughts on this?

Reply:
Hmm I imagine that we'd only want admins to have that privilege, or for no one to have that privilege at all... Not sure on what's the better choice

Reply:
I think I'd be good with admins having this ability!